librdfa-rdf4j/src/main/c/main.java

```diff
@@ -0,0 +1,82 @@
+
```

Comment:
Missing license header.

Reply:
I am sorry, this file was intended for testing purposes when I was testing the bridge interaction. I will deleted.